alembic/versions/0193b4725f32_bug_1132858_remove_extensions.py

```diff
@@ -0,0 +1,51 @@
+"""bug 1132858 remove extensions table
+
+Revision ID: 0193b4725f32
+Revises: bb8cdbb8a6bd
+Create Date: 2018-01-31 14:21:41.032179
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '0193b4725f32'
+down_revision = 'bb8cdbb8a6bd'
+
+from alembic import op
+from socorro.lib import citexttype, jsontype, buildtype
+from socorro.lib.migrations import fix_permissions, load_stored_proc
+
+import sqlalchemy as sa
+from sqlalchemy import types
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import table, column
+
+
+def upgrade():
+    # Get a list of ALL tables that start with 'extensions_*' and drop them
+    connection = op.get_bind()
+    cursor = connection.connection.cursor()
+    cursor.execute("""
+        SELECT c.relname FROM pg_catalog.pg_class c
+        WHERE c.relkind = 'r' AND c.relname LIKE 'extensions_%'
+    """)
+    all_table_names = []
+    for records in cursor.fetchall():
+        table_name, = records
+        all_table_names.append(table_name)
+
+    all_table_names.sort(reverse=True)
```

Comment:
Why the sort?

Reply:
So... it looks like the migration I copied this code from picked up `plugin_reports` in addition to the partitioned tables--`plugin_reports` had to be deleted last.

In this case, since my LIKE clause doesn't pick up `extensions`, we don't have the same issue. So we can nix this.